cypress/support/generate.js

```diff
@@ -0,0 +1,12 @@
+import { faker } from '@faker-js/faker';
+
+function generateUser() {
+  const randomNumber = Math.random().toString().slice(2, 6);
+  const username = faker.internet.userName() + '_' + randomNumber;
+  const email = `${username}@mail.com`;
+  const password = '12345Qwert!';
```

Comment:
The password is currently static, which could lead to problems if you need to generate multiple users. Consider generating a unique password for each user to avoid potential conflicts. You can use the `faker` library to generate a random, but valid password.